app/posts/apply-for-teacher-training/2020-01-16-question-protocol-for-pilot.md

```diff
@@ -0,0 +1,1722 @@
+    required: "No",
+    api: {
+      param: "work_experience.volunteering[].working_with_children",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
+    },
+    why: [{
+      who: ["candidate"],
+      what: "may use this to indicate experience working with children"
+    }, {
+      who: ["provider"],
+      what: "uses this to build a picture of a candidate’s suitability"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Reference may collaborate some of this information"],
+    notes: ["We’ve allowed people to include volunteering roles that don't involve kids – this could be clearer though."]
+  }
+}) }}
+
+## Training with a disability
+
+This question in the form, as described below, was removed from the initial pilot for launch. It will return once additional research in this area has been completed.
+
+Candidates can update their answers before (and 5 working days after) submitting their application.
+
+{{ appProtocol({
+  title: "✘ Candidate wants to disclose a disability",
+  items: {
+    question: "Do you want to disclose a disability?",
+    required: "Yes",
+    why: [{
+      who: ["service"],
+      what: "uses the answer to show/hide textarea"
+    }],
+    what: ["Answer not submitted as part of application"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "✘ Disability disclosure",
+  items: {
+    question: "Tell us about your disability",
+    required: "No",
+    api: {
+      param: "candidate.disability_disclosure",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "may use this to support the candidate at interview (and training if accepted onto course)"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Assessed at or before interview"],
+    notes: [
+      "Does a provider or someone else look at this and make any reasonable adjustments? Is there a process to work out what adjustments can be made or not? Research currently being undertaken."
+    ],
+    history: ["/apply-for-teacher-training/training-with-a-disability"]
+  }
+}) }}
+
+## Degrees
+
+Candidates can update their answers before (and 5 working days after) submitting their application.
+
+{{ appProtocol({
+  title: "Qualification - type",
+  items: {
+    question: "Type of degree",
+    required: "First degree: Yes
+      Additional degree(s): No",
+    api: {
+      param: "qualifications.degrees[].qualification_type",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "needs to ensure candidate holds a first degree of a UK higher education institution or equivalent qualification"
```

Comment:
Suggest linking to:
https://www.gov.uk/government/publications/initial-teacher-training-criteria/initial-teacher-training-itt-criteria-and-supporting-advice#c12-degree-criteria

Reply:
Done.